Bugzilla/Install/Filesystem.pm

```diff
@@ -481,14 +484,11 @@ sub update_filesystem {
     }
     elsif (-e 'index.html') {
         my $templatedir = bz_locations()->{'templatedir'};
-        print <<EOT;
-
-*** It appears that you still have an old index.html hanging around.
-    Either the contents of this file should be moved into a template and 
-    placed in the '$templatedir/en/custom' directory, or you should delete 
-    the file.
-
-EOT
+        print join("\n",
+            "*** It appears that you still have an old index.html hanging around.",
+            "Either the contents of this file should be moved into a template and ",
```

Comment:
Trailing spaces here will be appended by newline, which works out to be a whitespace error that git can't see.